packages/airnode-node/config/config.example.json

```diff
@@ -20,7 +20,8 @@
           "value": 3.12,
           "unit": "gwei"
         },
-        "fulfillmentGasLimit": 500000
+        "fulfillmentGasLimit": 500000,
+        "withdrawalRemainder": "0"
```

Comment:
Another option instead of strings throughout could be numbers with something like `withdrawalRemainderGwei`

Reply:
Isn't the standard way of specifying amounts
```
{
  "value": 3.12,
  "unit": "gwei"
}
```
?

Reply:
Yes- I've updated to use this approach.